UefiCpuPkg/Library/CpuExceptionHandlerLib/SecPeiCpuExceptionHandlerLib.inf

```diff
@@ -66,12 +66,9 @@
   CpuLib
 
 [Pcd]
-  gEfiMdeModulePkgTokenSpaceGuid.PcdCpuStackGuard
+  #gEfiMdeModulePkgTokenSpaceGuid.PcdCpuStackGuard  # MU_CHANGE
   gUefiCpuPkgTokenSpaceGuid.PcdCpuStackSwitchExceptionList
   gUefiCpuPkgTokenSpaceGuid.PcdCpuKnownGoodStackSize
 
-[FeaturePcd]

```

Comment:
In other places code was commented out versus deleted here. Would make for more consistent diffs if left but commented.